packages/core/src/components/Spinner/Spinner.scss

```diff
@@ -0,0 +1,186 @@
+@import '@cmsgov/design-system-support/src/settings/index';
+
+@mixin spinner-size($inner-size, $outer-size) {
+  height: $outer-size;
+  width: $outer-size;
+  $padding: (($outer-size - $inner-size) / 2);
+
+  &::before,
+  &::after {
+    border-width: ($inner-size / 8);
+    height: $inner-size;
+    left: $padding;
+    top: $padding;
+    width: $inner-size;
+  }
+}
+
+/*
+Spinner
+
+Spinners signify that the application is waiting for an asychronous operation to complete.
+
+.ds-c-spinner
```

Comment:
We could remove this. Devs can look at the code markup samples to find the necessary component class name. It's not currently information we include on other component documentation pages.